snippets/chunk-array.md

```diff
@@ -0,0 +1,14 @@
+### Chunk Array
+
+Creates an array of elements split into groups the length of size. 
+If array can't be split evenly, the final chunk will be the remaining elements.
```

Comment:
The description should describe the *how* of the implementation (i.e. how the code does what it does), not the *what* (i.e. what you aim to accomplish with this function). Please update your snippet.